.buildkite/postgres-config.yaml

```diff
@@ -1,16 +1,19 @@
+# Configuration file used for testing the 'synapse_port_db' script.
+# Tells the script to connect to the postgresql database that will be available in the
+# CI's Docker setup at the point where this file is considered.
 server_name: "test"
 
 report_stats: false
 
 database:
   name: "psycopg2"
   args:
-    user: postgres
-    host: postgres
-    password: postgres
-    database: synapse
+      user: postgres
```

Comment:
not a blocker, but the indentation here is inconsistent: are you using 2 spaces or 4 spaces?